airflow/jobs/local_task_job.py

```diff
@@ -103,16 +100,8 @@ def signal_handler(signum, frame):
 
                 # Periodically heartbeat so that the scheduler doesn't think this
                 # is a zombie
-                try:
-                    self.heartbeat()
-                    last_heartbeat_time = time.time()
-                except OperationalError:
-                    Stats.incr('local_task_job_heartbeat_failure', 1, 1)
```

Comment:
I guess this removes this metric - we should probably add a note about that to UPDATING.md, or add it back in in some fashion inside BaseJob.heartbeat?

Reply:
This metric is basically not useful at all since all exceptions are swallowed by `heartbeat()`. Yes let me make a more generic metrics inside `heartbeat()` as an replacement.